cpu/stm32_common/stm32_mem_lengths.mk

```diff
@@ -258,6 +258,16 @@ else ifeq ($(STM32_TYPE), L)
       RAM_LEN = 640K
     endif
   endif
+else ifeq ($(STM32_TYPE), W)
+  ifeq ($(STM32_FAMILY), B)
+    ifeq ($(STM32_MODEL), B55)
+      ifeq ($(STM32_ROMSIZE), C)
+        RAM_LEN = 128K
+      else ifneq (, $(filter $(STM32_ROMSIZE), E G))
+        RAM_LEN = 256K
```

Comment:
Are you sure of this value ? If I set it to 192K then tests/malloc is working otherwise it crashes badly.